### ISSUE TITLE
Some icons/screenshots of famous softwares are not displayed

### ISSUE THREAD

**Issue by tobiasBora** [snowfallorg/nix-software-center#57]
Thanks again for this nice software. Surprisingly, some icons/screenshots of famous softwares are not displayed, in my case firefox and chromium have no icons:

![image](https://github.com/snowfallorg/nix-software-center/assets/2164118/b5373478-0eaa-4d4d-9fa8-b0a12d9719e2)

Also I'm curious, how are icons/screenshots even obtained? Might interest also https://github.com/snowfallorg/nix-software-center/issues/53

**Comment by KiaraGrouwstra** [comment]
i see icons included as build input [here](https://github.com/snowfallorg/nix-software-center/blob/68e87fb6e24fb3e6fe65449918d0cbda96622b65/default.nix#L52) (`gnome.adwaita-icon-theme`)

**Comment by tobiasBora** [comment]
I don't think this is related, `adwaita-icon-theme` seems to only contains icons like "new file", "new folder" etc, not application-related icons.

**Comment by KiaraGrouwstra** [comment]
might it be this just uses system icons?

**Comment by tobiasBora** [comment]
Well each software installs the icons when they get install. I do believed it is obtained using the above AppStream stuff

**Comment by KiaraGrouwstra** [comment]
hm, as an app store it shows applications that have not yet been installed too right?

**Comment by tobiasBora** [comment]
Sure, that's precisely why we can't rely on system-installed icons

**Comment by KiaraGrouwstra** [comment]
hm, i was under the impression installed icon packs may well include icons for a [bunch of applications](https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/blob/master/README.md) not necessarily restrained to applications that have been installed.
personally, as a user i tried installing such a pack i found pretty, meaning i would actually prefer for it to be used.

**Comment by tobiasBora** [comment]
Well, I guess custom icon themes are ignored by this program, but to be honest I have no idea. In any case, even if the icon does not follow the theme, it will only be show in nix-software-center, but after installation, you will of course see the custom theme in your desktop list of applications etc. If you still want to use your theme icon pack for nix-software-center, I guess you want to fill a separate bug.
